src/tests/tablet/schedule/AMeetingCannotBeUpdatedWhenItHasConflictsWithOtherMeeting.java

```diff
@@ -0,0 +1,89 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import framework.common.MeetingMethods;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC12: Verify an existing meeting cannot be updated if it has conflicts with another meeting
+ * with another meeting
+ * @author Asael Calizaya
+ *
+ */
+public class AMeetingCannotBeUpdatedWhenItHasConflictsWithOtherMeeting {
+	private ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	private List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");
+	private String nameMeeting1 = meetingData.get(0).get("Subject");
+	private String organizer1 = meetingData.get(0).get("Organizer");
+	private String password1 = meetingData.get(0).get("Password");
+	
+	private String nameMeeting2 = meetingData.get(1).get("Subject");
+	private String organizer2 = meetingData.get(1).get("Organizer");	
+	private String password2 = meetingData.get(1).get("Password");
+	
+	private String roomName = meetingData.get(1).get("Room");
+	private String authentication1 = organizer1 + ":" + password1;
+	private String authentication2 = organizer2 + ":" + password2;
+
+	@BeforeMethod(groups = "ACCEPTANCE")
+	public void creationMeetingsPreCondition() {
+		MeetingMethods preCondition = new MeetingMethods();
+
+		//Meeting 01
+		preCondition.createMeetingFromHome(meetingData.get(0).get("Organizer"),
```

Comment:
you should be persistent in your TCs i.e. if you decide use this structure - > maintain this
String organizer = meetingData.get(0).get("Organizer")
createMeeting(..,organizer,...)